src/main/java/com/google/codeu/data/Message.java

```diff
@@ -18,14 +18,28 @@
 
 import java.util.UUID;
 
-/** A single message posted by a user. */
+/** The sting recipient helps to display messages received from another user. */
 public class Message {
 
+  private String recipient;
```

Comment:
This belongs together with the other private member variables (id, user, text, timestamp) on lines 37-40.